modules/navier_stokes/doc/content/modules/navier_stokes/sqa/navier_stokes_srs.md

```diff
@@ -1 +1,49 @@
 !template load file=sqa/module_srs.md.template category=navier_stokes module=Navier Stokes
+
+!template! item key=system-scope
+!! system-scope-begin
+The [!ac](MOOSE) Navier Stokes module provides numerical discretizations of the
+Navier Stokes equations to model the flow of fluid through regular and porous media.
+The equations discretized include the conservation of mass, momentum, energy and of transported
+scalars / species.
+It covers a wide range of fluids and of fluid flow regimes. It covers both natural and forced
+convection regime, and should be able to model conjugate heat transfer between the fluid
+and solid phases. A number of scalar species can be transported using the velocity
+field determined from the fluid flow equations.
+It can be used as a standalone application or can be included in
+downstream applications interested in modeling multiphysics problems involving fluid flow.
+!! system-scope-finish
+!template-end!
+
+!template! item key=system-purpose
+!! system-purpose-begin
+The purpose of this software is to allow simulation of fluid flow in regular and porous media.
+It should be able to determine the pressure field and velocity fields in the fluid domain, as well
+as the temperature field in the case of conjugate heat transfer simulations. It aims to also be able to
+transport scalar species in the flow fields determined.
+
+This software is intended as a coarse mesh computational fluid dynamics simulation tool and to be the
+numerical backbone of the Pronghorn software. Its goals include the simulation of coolant flow in
```

Comment:
I might delete this sentence. I think it is unnecessarily narrowing. In the very short term the module will also be the backbone of the SAM multi-D capability. Moreover, though the scope for NEAMS is coarse mesh, I am hopeful that if not now, then some day it can also be used for finer meshes as well, e.g. when we have scalable solvers

Reply:
We could update when the SAM multi-D is merged in.
Similarly once we have shown we can do finer meshes in some capacity we can update.

but actually are these documents updateable when needed @cticenhour ?

Reply:
These documents are living ones, meant to be kept up to date as the code they're associated with change. I tend toward more broad statements and refer back to design documentation for specifics myself, but I leave those sorts of things to the module maintainers for how they want to outline the contents of their code.

Reply:
removed. We ll update after SAM merge